docs/proposals/202101_endpoint_discovery.md

```diff
@@ -42,20 +42,25 @@ info:
     blah: ...
 ```
 
-Right now this info API would be added to existing components Ruler, Store, Sidecar, Compact, Downsample, Receive. But this would be added in the future for Targets, Exemplers, etc.
+Right now this info service would be added to existing components Ruler, Store, Sidecar, Compact, Downsample, Receive.
```

Comment:
`bucket downsample` doesn't expose anything over gRPC now (and probably will never do that) so I think we shouldn't mention it here: 
```suggestion
Right now this Info service would be added to existing components Ruler, Store, Sidecar, Compact, Receive.
```

Reply:
I have the same question. How do we expose gRPC on compact? For what APIs?

Reply:
Yeah, I think this was just a slight oversight on our part, there's no plan to have a gRPC API on Compactor yet.

Reply:
yes agree with @onprem, It will not affect any component not exposing anything over gRPC. we will remove both compact and downsample.

Reply:
And we are missing Querier here too, as it also exposes a Store API we should add Info service there as well. Maybe not mentioning the exact component names will be better.
```suggestion
Right now this Info service would be added to existing components exposing a gRPC server.
```